### PR TITLE
Add internal solution for Three Days Ago (abc295_d)

### LINKED ISSUE #3730: Three Days Ago

### PR DESCRIPTION
**Pull request by ChuangSheep**
Closes #3730 

_If any of the below don't apply to this Pull Request, mark the checkbox as done._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions).
  - I understand that if it is clear that I have not attempted to follow these conventions, my PR will be closed.
  - If changes are requested, I will re-request a review after addressing them.

### CODE REVIEW

File: solutions/silver/ac-theedaysago.mdx [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+	# update the parity of the current digit
+	cur_state ^= 1 << int(digit)
+	# update the counter for the state
+	states[cur_state] += 1
+
+ans = 0
+for ct in states:
+	ans += ct * (ct - 1) / 2
+
+print(int(ans))
```
Comment (SansPapyrus683) [inline solutions/silver/ac-theedaysago.mdx:123]:
```suggestion
print(ans)
```

**Review by SansPapyrus683 — APPROVED** [review]
gotta love it

**Review by envyaims — APPROVED** [review]
lmao holy crap these problems have only been on for half a week and everyone is already requesting sols

**Comment by envyaims** [comment]
LMAO I SPELLED THREE WRONG IN THE ID YOU CAN CHANGE IT IF YOU WANT

**Comment by SansPapyrus683** [comment]
💀

File: solutions/silver/ac-theedaysago.mdx (hunk 1 above)
Comment (SansPapyrus683) [reply solutions/silver/ac-theedaysago.mdx:123]: use // for the division then

**Comment by SansPapyrus683** [comment]
change the problem id (fix typo) & i'll merge

**Pull request merged**